site-scrapers/MyChartAPI.js

```diff
@@ -108,14 +108,17 @@ module.exports.UpdateResults = function UpdateResults(results, postResponse) {
     for (const dayEntryProperty in allDays) {
         if (Object.prototype.hasOwnProperty.call(allDays, dayEntryProperty)) {
             dayEntry = allDays[dayEntryProperty];
-            const date = new Date(dayEntry.DateISO);
+            //force midnight local time zone to avoid UTC dateline issues
+            const date = new Date(`${dayEntry.DateISO}T00:00`);
             const slots = dayEntry.Slots;
             const slotsAvailable = slots.length || 0;
-
-            if (date >= new Date()) {
+            let midnightToday = new Date();
+            midnightToday.setHours(0, 0, 0, 0);
+            if (date >= midnightToday) {
                 results.availability[
-                    `${date.getMonth() + 1}/${date.getDate() +
-                        1}/${date.getFullYear()}`
+                    `${
+                        date.getMonth() + 1
+                    }/${date.getDate()}/${date.getFullYear()}`
```

Comment:
We don't need to carry forward the timezone information into the UI? We're just going to return something like "02/16/2021", which is still subject to UTC transformation, no?

Reply:
This is probably fine now, but maybe we centralize this date formatting so we can use it on all scrapers?

Reply:
@adpjay We're sending a string to the UI so there's no date transformation once it's in our `availability` object.
@zpeyton I've been thinking about something similar - ensuring the return object is in an acceptable format. I'll open a new issue and list both my idea and your point as well.